src/utils/emasser/outputFormatter.ts

```diff
@@ -67,102 +71,144 @@ export function outputFormat(data: object, doConversion = true): string {
       formatDataObj = _.get(data, 'data') || {}
     }
 
+    const paginationObj = {pagination: _.get(formatDataObj, 'pagination')}
```

Comment:
Is there a guarantee that the passed in `data` object will contain a `pagination` field?

Reply:
Not every data object will have a pagination child object, however if the pagination content is not extracted here, before any data manipulation, the _.get doe not extrat the data correctely. I will move the code into the test doConversion.